doc/distrib/NodeHelpFiles/_review/Revit.Elements.Family.ByName.md

```diff
@@ -0,0 +1,8 @@
+## In Depth
+`Family.ByName` returns the family definition from the current document (if available). If the family is not available in the current document, a null value is returned.
+
```

Comment:
For clarity, might be good to add a note to mention what the id next to the family refers to.